File: homework/08/03.py
def classify_even_odd(numbers):
    result = {
        'even': [],
        'odd': []
    }
    for num in numbers:
        if num % 2 == 0:
            result['even'].append(num)
        else:
            result['odd'].append(num)
    return result
  
print(classify_even_odd([1, 2, 3, 4, 5, 6]))
print(classify_even_odd([-3, -2, -1, 0, 1 ,2 ,3]))
print(classify_even_odd([]))

File: homework/03/03.js
/* 陣列操作

建立一個陣列 numbers = [3, 7, 1, 9, 4]。
將陣列中的數字由小到大排序，並輸出結果。
可以直接用 sort 函數 */
let numbers = [3, 7, 1, 9, 4]

numbers.sort((a, b) => a - b)
console.log(numbers)
